Multisig/UI/Safe Management/Create Safe/GoogleWeb3AuthLoginModel.swift

```diff
@@ -33,9 +28,6 @@ class GoogleWeb3AuthLoginModel {
                                    subVerifierDetails: [sub],
                                    network: .CYAN
             )
-            await MainActor.run(body: {
```

Comment:
Are you sure this is not needed anymore?, then why it was there in the original place?

Reply:
I think it worked by accident. :-/